src/bench/logging.cpp

```diff
@@ -1,64 +0,0 @@
-// Copyright (c) 2020-2022 The Bitcoin Core developers
```

Comment:
Not sure about removing the logging benchmarks.  Logging can be very high frequency. Even if that is more likely to occur for developers, it may be useful to be able to track this.